src/Language/Haskell/Refact/Refactoring/GenApplicative.hs

```diff
@@ -0,0 +1,383 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE DeriveDataTypeable #-}
+module Language.Haskell.Refact.Refactoring.GenApplicative where
```

Comment:
Please export only the two entrypoints for the refactoring, and rename `comp` to `compGenApplicative`, to match the other refactorings